radiography/src/main/java/radiography/FocusedWindowViewFilter.kt

```diff
@@ -8,6 +8,6 @@ import android.view.View
 object FocusedWindowViewFilter : ViewFilter {
 
   override fun matches(view: Any): Boolean {
-    return view is View && (view.parent?.parent != null || view.hasWindowFocus())
+    return view !is View || (view.parent?.parent != null || view.hasWindowFocus())
```

Comment:
I'm curious why this over the && ?

Reply:
If a thing is not a `View`, this filter should ignore it. We need to return `true` in that case, so that other filters can still process the value.

Reply:
This should be more clear once we decide on an answer to the above comment.